modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -93,11 +93,21 @@ def header_md5
         def source_data
           {
             name: source_name,
-            icn: current_user.icn,
+            icn: nullable_icn,
```

Comment:
As far as I can tell, icn is only used within the PoaUpdater job which is not called if headers for a Veteran are passed in
https://github.com/department-of-veterans-affairs/vets-api/blob/49b0e6c5683a9e6e614c7759dc88a273891f3c4f/modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb#L40

And, if icn is needed, it looks like the application defaults to a setting if icn isnt present
https://github.com/department-of-veterans-affairs/vets-api/blob/49b0e6c5683a9e6e614c7759dc88a273891f3c4f/modules/claims_api/app/models/claims_api/power_of_attorney.rb#L130